adam-core/src/test/scala/org/bdgenomics/adam/rdd/read/ADAMAlignmentRecordContextSuite.scala

```diff
@@ -152,7 +172,7 @@ class ADAMAlignmentRecordContextSuite extends SparkFunSuite {
    same location as createTempFile() uses, so therefore the returned path from this method
    should be suitable for adamParquetSave().
    */
-  def tempLocation(suffix: String = ".adam"): String = {
+  def tempLocation(suffix: String = "adam"): String = {
```

Comment:
This change is what caused the unit test failure; IMO, it should probably go back to the old version.